main.cpp

```diff
@@ -199,11 +205,20 @@ struct Truck
 
 Truck::Truck()
 {
+    speedLimit = 100;
+    weight = 400;
     std::cout << "'Truck' created" << std::endl;
 }
 
+Truck::Truck(int maxSpeed, int truckWeight)
+{
+    speedLimit = maxSpeed;
+    weight = truckWeight;
+}
```

Comment:
Delete this constructor. 
You'll learn about Overloading constructors and functions later in the course.
- [ ] remove this.